src/util.test.ts

```diff
@@ -84,15 +85,21 @@ describe('util', () => {
       expect(result).toEqual([
         {
           profile: {
-            scope: 'scope',
-            name: 'name',
+            id: {
+              id: 'scope/name',
+              scope: 'scope',
+              name: 'name',
+            },
```

Comment:
You shouldn't change result shape.
This change breaks anything using this function. eg. [validate](https://github.com/superfaceai/station/blob/main/src/validate.test.ts#L16)

Reply:
This is just the format of profile `check` function from `@superfaceai/cli` accepts.

Reply:
But right the profile format from `@superfaceai/cli` is complex and we can map to it before `check` function is called.

Reply:
No no, you have it right. I overlooked https://github.com/superfaceai/station/pull/82/files#diff-017358cdc577c3a0f1d5f0d4a96fb167e4a4d1705958540e6a2bd6770873ed6dR17